src/imgui/ImGuiReverseBar.cc

```diff
@@ -139,7 +153,7 @@ void ImGuiReverseBar::showMenu(MSXMotherBoard* motherBoard)
 			}
 		});
 		if (ImGui::MenuItem("Open savestates folder...")) {
-			SDL_OpenURL(strCat("file://", FileOperations::getUserOpenMSXDir(), "/savestates").c_str());
+			SDL_OpenURL(strCat("file://", FileOperations::getUserOpenMSXDir(), "/", STATE_DIR).c_str());
```

Comment:
Very much a detail: replace "/" with '/'  (when appending sub-strings that have length=1).

Reply:
In the 1st commit you indeed resolve this, but then in your 2nd commit you accidentally revert it.

Reply:
Oops, that happened during a rebase that had conflicts and somehow I got it partially reverted :( Should be fixed now.